src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,104 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int LENGTH = 10;
+
+    private T [] defaultArray;
+    private int size;
+
+    public ArrayList() {
+        this.defaultArray = (T[]) new Object[LENGTH];
+        this.size = 0;
+    }
+
+    private boolean isFull() {
+        if (size >= defaultArray.length) {
+            T [] largeArray = (T []) new Object [(int)(defaultArray.length * 1.5)];
+            System.arraycopy(defaultArray, 0, largeArray, 0, size);
+            defaultArray = largeArray;
+        }
+        return false;
```

Comment:
is this result informative? let's make this method return void

Reply:
Fixed